docs/source/blueprints/BP003.rst

```diff
@@ -0,0 +1,558 @@
+BP003
+#####
+
+:Number: BP003
+:Title: N(ext)Runner Task Life-Cycle
+:Author: Cleber Rosa <crosa@redhat.com>
+:Discussions-To: avocado-devel@redhat.com
+:Reviewers:
+:Created: 20-July-2020
+:Type: Architecture Blueprint
+:Status: Draft
```

Comment:
So I guess you mean, anticipating that it'd be accepted, this should now contain `WIP` so that another PR version changing just that is not necessary?

Reply:
yes. We need to improve this yet, but yes.

For instance, we could let the bp in draft for a while inside the repo, but this is not the case.